lib/install/management/network.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package management
+
+import (
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/vmware/vic/lib/metadata"
+)
+
+func (d *Dispatcher) removeNetwork(conf *metadata.VirtualContainerHostConfigSpec) error {
+	if !conf.CreateBridgeNetwork {
+		log.Infof("Bridge network is not created for this VCH, leave it there.")
+		return nil
+	}
+
+	if d.session.Host == nil {
+		log.Warnf("Remove network is not supported in VC")
+		return nil
+	}
+	log.Infof("Removing Portgroup")
```

Comment:
Add the name of the switch in here so that if it fails a viadmin knows what to manually delete.